containers/daap-athena-load/src/var/task/athena_load_handler.py

```diff
@@ -26,12 +27,15 @@ def handler(
     extraction = RawDataExtraction.parse_from_uri(full_s3_path)
     data_product_element = extraction.element
 
+    data_product_name = data_product_element.curated_data_table.database
```

Comment:
We now need to distinguish between the data_product_name and the database name that has a version suffix, otherwise you'll get a schema doesn;t exist error, e.g. looking for `{data_product_name}_v1`
```suggestion
    data_product_name = data_product_element.data_product.name
    database_name = data_product_element.curated_data_table.database